examples/games/breakout.rs

```diff
@@ -101,15 +98,16 @@ struct Brick;
 #[derive(Resource, Deref)]
 struct CollisionSound(Handle<AudioSource>);
 
-// This bundle is a collection of the components that define a "wall" in our game
-#[derive(Bundle)]
-struct WallBundle {
-    // You can nest bundles inside of other bundles like this
-    // Allowing you to compose their functionality
-    sprite: Sprite,
-    transform: Transform,
-    collider: Collider,
-}
+// Default is required to be part of the Required Components used with the Wall Component bellow
+#[derive(Component, Default)]
+struct Collider;
+
+// This is a collection of the components that define a "Wall" in our game.
+// from bevy version 0.15 onward the main way has shifted from using a Bundle, to Required Components
+// Allowing you to compose their functionality, See https://bevyengine.org/news/bevy-0-15/#required-components
```

Comment:
I find it strange to comment on Bevy versions or link to release notes. Examples should demonstrate *current* functionality and APIs, not describe the history of how the API or conventions have changed.

Reply:
I agree with that. In my mind I wanted to mention the shift from bundles since they are not being deprecated and still in the system, so wanted to highlight hem as "another way" of combining components, but moving forward they are not the main way.

I will push an update and, as you said, keeping it to the point of how the current API works.